docs/tutorials/morpheusvm/4_deploying_vm.md

```diff
@@ -361,13 +361,12 @@ and the RPC endpoint. We can do this by executing the following commands:
 ./hypersdk-cli key set --key=0x323b1d8f4eed5f0da9da93071b034f2dce9d2d22692c172f3cb252a64ddfafd01b057de320297c29ad0c1f589ea216869cf1938d88c9fbd70d6748323dbf2fa7
 ```
 
-Your command line should look as follows:
+You should see the following outputs:
 
 ```bash
-AVL-0W5L7Y:tutorial rodrigo.villar$ ./hypersdk-cli endpoint set --endpoint=http://localhost:9650/ext/bc/tutorial/ 
-Endpoint set to: http://localhost:9650/ext/bc/tutorial/
-AVL-0W5L7Y:tutorial rodrigo.villar$ ./hypersdk-cli key set --key=0x323b1d8f4eed5f0da9da93071b034f2dce9d2d22692c172f3cb252a64ddfafd01b057de320297c29ad0c1f589ea216869cf1938d88c9fbd70d6748323dbf2fa7
-✅ Key added successfully!
```

Comment:
Could we show output for each line one by one? It is confusing imo to show command 1, 2, and 3 and then show the output of all of them afterwards

Reply:
Done